bytecode/cfg.py

```diff
@@ -61,32 +70,23 @@ def _compute_stack_size(block, size, maxsize):
         if isinstance(instr, SetLineno):
             continue
 
-        size += instr.stack_effect
-        maxsize = max(maxsize, size)
-
-        if size < 0:
-            msg = 'Failed to compute stacksize, got negative size'
-            raise RuntimeError(msg)
-
         if instr.has_jump():
+            # first compute the taken-jump path
+            before_size = size
+            update_size(instr.stack_effect(1))
```

Comment:
It would be better to use boolean value `True`. Or even with keyword argument: `jump=True`.